bin/rad-diff

```diff
@@ -207,41 +199,32 @@
     (def mk-key
       (fn [v]
         (match v
-          (/as 'a {:activity :new-diff
-                   :diff     'diff})
-            [(string-append "[new/changed] " (pretty-headline diff)) a]
-          (/as 'a {:activity :new-comments
-                   :diff     'diff})
-            [(string-append "[new comments] " (pretty-headline diff)) a]
-          (/as 'i {:diff 'diff})
+          (/as 'i 'diff)
```

Comment:
Now you can remove the `match` clause completely
~~~clojure
(fn [diff] [(pretty-headline diff) diff)])
~~~